samwise.js

```diff
@@ -1,6 +1,7 @@
 /**
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.